include/jsbsim_bridge.h

```diff
@@ -59,15 +60,16 @@ static constexpr int kDefaultSITLTcpPort = 4560;
 
 class JSBSimBridge {
  public:
-  JSBSimBridge(JSBSim::FGFDMExec *fdmexec, std::string &path);
+  JSBSimBridge(JSBSim::FGFDMExec *fdmexec, ConfigurationParser *cfg);
```

Comment:
Rather use refs here too:
```suggestion
  JSBSimBridge(JSBSim::FGFDMExec &fdmexec, ConfigurationParser &cfg);
```